plugins/include/sdktools_stocks.inc

```
@@ -73,3 +73,33 @@ stock int FindTeamByName(const char[] name)
 
 	return found_team;
 }
+
+/**
+ * Get the end point coordinates where client aim at.
+ *
+ * @param client			Player Index.
+ * @param pos				Destination vector to store the end point coordinates.
+ * @param flags				Trace flags. MASK_SHOT by default.
+ */
+stock void TraceRayClientAimEndPoint(int client, float pos[3], int flags = MASK_SHOT)
+{
+
+	if( !(0 < client <= MaxClients) || !IsClientInGame(client))
+		return;

```

This is just a minor nit but instead of just returning because the client isn't valid we should throw an error or remove these checks. The developer using this stock should add these checks before even thinking about using this stock.

---

If the author of the pr change the function to return a bool, it'd be ok to just return false here, or not check it at all.